cli/flags.go

```diff
@@ -56,7 +56,6 @@ func Init() (ret *Flags, err error) {
 	// takes input from stdin if it exists, otherwise takes input from args (the last argument)
 	if hasStdin {
 		if message, err = readStdin(); err != nil {
-			err = errors.New("error: could not read from stdin")
```

Comment:
err = fmt.Errorf("error: could not read from stdin: %w", err)

Reply:
This is already covered in `readStdin` that wraps it: https://github.com/ALX99/fabric/blob/7cbd49375a8429847750d060bceb4e090dfe341f/cli/flags.go#L81